### DISCUSSION TITLE
CopyQ : will commands also be automatically updated ?

### DISCUSSION THREAD

**Discussion by Clemens-Ratte-Polle** [hluk/CopyQ#2007] [General]
Moin (Hi in Plattdeutsch-German :) )
When updating CopyY will all commands also be updated?
Or do we have do download them on which place?

Some will have made changes to commands - how do updates preserve them?
thx

**Comment by dbareis** [comment]
On Windows I've never had my commands wiped up upgrades if that is what you are worried about.  You should be backing up your computer and include at least the copyq configuration.

In [github discussions](https://github.com/hluk/CopyQ/discussions/2004), I recently posted some code that will export commands into individual ini files, I have also written but not posted import of these.
